README.md

```diff
@@ -84,10 +84,12 @@ The thsvsCmdPipelineBarrier and thsvsCmdWaitEvents commands allocate
 temporary storage for the Vulkan barrier equivalents in order to pass them
 to the respective Vulkan commands.
 
-These use the `temp_alloc(size)` and `temp_free(x)` macros, which are by
-default set to `alloca(size)` and nothing, respectively.
+These use the `THSVS_TEMP_ALLOC(size)` and `THSVS_TEMP_FREE(x)` macros,
+which are by default set to alloca(size) and (0), respectively.
```

Comment:
This (and the header comment) should probably say ‘((void)(x))’ instead to match the implementation change?

Reply:
Done. Nice catch!